### ISSUE TITLE
Chromaticraft underground chambers are unbreakable to RoC boring machine

### ISSUE THREAD

**Issue by Kowalskicore** [ReikaKalseki/Reika_Mods_Issues#2077]
This is probably intended behaviour, but I'm reporting it regardless since it kind of annoys me.

Basically, the underground chambers containing what WAILA calls "crystal.pylon" are indestructible to boring machines no matter the power input. Even 2.5 petawatts produced by creative industrial coils is not enough.

https://i.imgur.com/Sd3qtUs.png - boring machine recieving 2.517 PW.
https://i.imgur.com/KgWSsW7.png - underground chamber, it won't budge.

Previously I just teleported myself to the borer head and destroyed the crystal pylon thing in creative mode, which allowed the borers to easily munch through the chamber. Removing the mod from the pack would be an easy "fix", but I added it for a friend, and all the missing blocks would just be weird. Perhaps a config option could be added (if it doesn't already exist) that turns off the block-breaking protection?

**Comment by voidsong-dragonfly** [comment]
Run the machines lower. Running them just above bedrock means you will never hit one.

**Comment by Kowalskicore** [comment]
But that doesn't look half as impressive as a large array of boring machines drilling a massive tunnel into the earth.

**Comment by voidsong-dragonfly** [comment]
True. I normaly have two stacked on top of the bedrock (5-15 ish)and one for certus+fluorite

**Comment by Kowalskicore** [comment]
I run boring machines at those heights too, and still encounter chambers. It is a rare issue I must admit, but bothersome nonetheless.

**Comment by Barhandar** [comment]
It would make some out-of-game sense to have chambers and their pylons be only unbreakable if the player doesn't have relevant progression.

**Comment by ReikaKalseki** [comment]
At the point of encountering the wall of a structure, the borer does not know that it *is* a structure, let alone what type. All it knows is that it hit some blocks with negative hardness (ie are unbreakable). Making it able to bypass that will make it able to mine through *everything* unbreakable.

**Comment by Kowalskicore** [comment]
Oh, okay. If there's no way around it, guess I'll keep going as I was then.

**Comment by chotke** [comment]
I have a same problem, avr 1 shrine per 1700 blocks at Y=15. Remove chromaticraft. There is no other way.

**Comment by ReikaKalseki** [comment]
Or set them up in another dimension.

**Comment by voidsong-dragonfly** [comment]
Unless the Deep Dark has no shrines, MystCraft is the only way to do so. RFTools dimensions that inherit overworld features enough to have oregen have also have the CrC shrines. And the Aroma mining dimension has the shrines as well. Which reminds me, I need to add mystcraft to my pack for that reason.

**Comment by ReikaKalseki** [comment]
> Unless the Deep Dark has no shrines

It has none.

**Comment by HonkerBonker** [comment]
> And the Aroma mining dimension has the shrines as well. 
    I:"Structure Dimension Blacklist" <
        6
     >
- that added to chromaticraft config seem to stopped CrC gen in aroma dimension . 6 is in aroma config for its world id.
